src/halmos/cheatcodes.py

```diff
@@ -230,6 +230,19 @@ def symbolic_storage(ex, arg, sevm, stack, step_id):
     return ByteVec()  # empty return data
 
 
+def snapshot_storage(ex, arg, sevm, stack, step_id):
+    account = uint160(arg.get_word(4))
+    account_alias = sevm.resolve_address_alias(
+        ex, account, stack, step_id, allow_branching=False
+    )
+
+    if account_alias is None:
+        error_msg = f"snapshotStorage() is not allowed for a nonexistent account: {hexify(account)}"
+        raise HalmosException(error_msg)
+
+    return ByteVec(uint256(ex.storage[account_alias].cnt))
```

Comment:
if cnt is an int, might as well stay concrete:

```suggestion
    return ByteVec(int.to_bytes(ex.storage[account_alias].cnt, length=32))
```